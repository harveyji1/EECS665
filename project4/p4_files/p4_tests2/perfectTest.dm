a : perfect int;
flag: bool;
