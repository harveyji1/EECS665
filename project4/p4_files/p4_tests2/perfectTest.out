a{perfectint} : perfect int;
flag{bool} : bool;
