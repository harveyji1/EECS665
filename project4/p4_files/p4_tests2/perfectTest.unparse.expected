a{perfect int} : perfect int;
flag{bool} : bool;
